.dockerignore

```diff
@@ -0,0 +1,19 @@
+# Dependencies
+node_modules
+**/*/node_modules
+
+# coverage
+*/**/coverage
+*/**/coverageEnv
+*/**/coverage.json
+*/**/allFiredEvents
+*/**/scTopics
+
+
+apps/**/artifact.json
+future-apps/**/artifact.json
+
+# lerna
+lerna-debug.log
+
+# truffle build artifacts
```

Comment:
Can not be `**/*/build` added here too?
What about `.git`?

Reply:
good idea

Reply:
done